configs/11.0/packages/gcc/sources

```diff
@@ -32,9 +32,11 @@ ATSRC_PACKAGE_RELFIXES=
 ATSRC_PACKAGE_NAMESUFFIX="[C, C++ (g++), fortran, Go]"
 ATSRC_PACKAGE_STR_VER="${ATSRC_PACKAGE_NAME} ${ATSRC_PACKAGE_VER}-ibm-r${ATSRC_PACKAGE_REV}"
 ATSRC_PACKAGE_PRE="test -d gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
-ATSRC_PACKAGE_CO=([0]="svn co --revision ${ATSRC_PACKAGE_REV} svn://gcc.gnu.org/svn/gcc/branches/ibm/gcc-7-branch" \
-		  [1]="svn co --revision ${ATSRC_PACKAGE_REV} http://gcc.gnu.org/svn/gcc/branches/ibm/gcc-7-branch")
-ATSRC_PACKAGE_POST="mv gcc-7-branch gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}"
+ATSRC_PACKAGE_CO=([0]="git clone git://gcc.gnu.org/git/gcc.git")
+ATSRC_PACKAGE_GIT=([0]="git config --local remote.origin.fetch +refs/vendors/ibm/heads/*:refs/remotes/origin/ibm/*" \
+                   [1]="git fetch origin" \
+                   [2]="git checkout -b gcc-${ATSRC_PACKAGE_VER}${ATSRC_PACKAGE_REV:+-${ATSRC_PACKAGE_REV}} ${ATSRC_PACKAGE_REV}")
```

Comment:
@mscastanho In the past, it had to change the values frequently for tests (including removing the value of the revision) and we did not want to rewrite the entire file.
So, we ended up writing all of them in way that would allow to remove the revision and continue working.

Reply:
@tuliom Ok then. But in that case, shouldn't both strings be declared that? If we remove the revision, `gcc-${ATSRC_PACKAGE_REV:+-${ATSRC_PACKAGE_REV}}` will evaluate to something like `gcc-9`, while `gcc-${ATSRC_PACKAGE_VER}-${ATSRC_PACKAGE_REV}`, used to rename the directory in the next line, will be `gcc-9-`.

Reply:
@mscastanho I agree we could rework all `configs/*/packages/*/sources` files to have something lighter and homogeneous, but it's not the goal of the current patch ;)